flytectl/cmd/delete/delete.go

```diff
@@ -30,9 +32,18 @@ func RemoteDeleteCommand() *cobra.Command {
 		Short: deleteCmdShort,
 		Long:  deleteCmdLong,
 	}
+	var force bool
+	deleteCmd.PersistentFlags().BoolVarP(&force, "force", "f", false, "Force deletion without confirmation")
```

Comment:
I'm unfamiliar if there are any examples, but would be nice if just needed to include the force flag. 

Command could be:

 flytectl delete execution kxd1i72850  -d development  -p flytesnacks --force